scripts/run_backend_tests.sh

```diff
@@ -85,6 +85,10 @@ for arg in "$@"; do
   fi
 done
 
+# Compile typescript files
+echo "Compiling typescript..."
+$NODE_MODULE_DIR/typescript/bin/tsc
```

Comment:
Doesn't this need options (e.g. at least the dirname)? Ditto for other uses of tsc.

Reply:
Since we need to compile all ts files here and not some selected files, no arguments are needed. The compiler defaults to compiling all ts files in the containing directory of `tsconfig.json` except the ones specified through exclude option in `tsconfig.json`